lib/ReactViews/Analytics/BooleanParameterEditor.jsx

```diff
@@ -11,31 +12,47 @@ const BooleanParameterEditor = React.createClass({
     },
 
     onClick() {
-        const value = this.props.previewed.parameterValues[this.props.parameter.id];
-        this.props.previewed.setParameterValue(this.props.parameter.id, !value);
+        this.props.parameter.value = !this.props.parameter.value;
+    },
+
+    renderCheckbox() {
+        const value = this.props.parameter.value;
+        const name = this.props.parameter.name;
+        const description = this.props.parameter.description;
+
+        return (
+            <div>
+                <button type='button'
+                        className={Styles.btnRadio}
+                        title={description}
+                        onClick={this.onClick}>
+                    {value && <Icon glyph={Icon.GLYPHS.checkboxOn}/>}
+                    {!value && <Icon glyph={Icon.GLYPHS.checkboxOff}/>}
+                    {name}
+                </button>
+            </div>
+        );
     },
 
     renderRadio(state) {
         let name;
         let description;
-        let classNames;
-        const value = this.props.previewed.parameterValues[this.props.parameter.id];
+        const value = this.props.parameter.value === state;
```

Comment:
I might have the wrong idea here, but if this.props.parameter.value is false, value is false for both states, isn't it? This is only relevant if the value of this.props.parameter.value can be false; I'm assuming this can pick up a default value of false from somewhere.